freedomofpress/securedrop
# Removes duplicate tor-apt repo from apt config

## Status

Ready for review.

## Description of Changes

Fixes #2938.

Achieves the removal by renaming the file inside the `securedrop-config`
package. We can trust dpkg to remove the file once the package has been
upgraded. The Ansible logic was already using the filepath now used also
by the `securedrop-config` package, so no Ansible changes are required.

## Testing

1. Create fresh staging VMs on `develop`.
2. Check out this feature branch. 
3. Run `./testinfra/test.py app-staging` and confirm you get 1 failing test (for too many apt repos).
4. Run `make build-debs && vagrant provision /stag/`. 
5. Re-run `./testinfra/test.py app-staging` and confirm **all tests pass.**
## Deployment

Any special considerations for deployment? Consider both:

Yes, designed to clean up existing installs with a duplicate apt repo. Particularly if we consolidate the apt repos down the road, we'll want to make sure we don't leave any dangling references. The changes here should suffice.

## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you made changes to documentation:
